Pipfile

```diff
@@ -11,6 +11,7 @@ python-telegram-bot = "==12.2.0"
 sh = "*"
 readerwriterlock = "*"
 pyyaml = "*"
+numpy = "*"
```

Comment:
delete this dependency
just remove it from Pipfile and then do `pipenv install` to update Pipfile.lock